### PR TITLE
Upgrade V8 to 8.8.278.2

### PR DESCRIPTION
**Pull request by bnoordhuis**

### CODE REVIEW

**Comment by bartlomieju** [comment]
@bnoordhuis rebase on `master` to get CI running again

**Comment by bnoordhuis** [comment]
The aarch64-unknown-linux-gnu runner failed with this:

> Error: thread 'terminate_execution' panicked at 'called `Option::unwrap()` on a `None` value', src/context.rs:28:6

Which is a `let context = v8::Context::new(scope);` statement. Re-running to see if it's a flake or not.

**Comment by bnoordhuis** [comment]
This time it failed at the build step with sccache errors...
```
sccache: encountered fatal error
  sccache: error : corrupt deflate stream
  sccache:  cause: corrupt deflate stream
```
One more time...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
           # come from the registry. By additionally saving the sccache cache
           # directory it's still possible to build v8 fast.
           path: |-
-            target/sccache
+            #target/sccache
```
Comment (bartlomieju) [inline .github/workflows/ci.yml:107]: Revert before merge?

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**